src/core/getEASAttestations.ts

```diff
@@ -1,9 +1,14 @@
+import { createEasGraphQLClient } from '../network/easGraphQL';
+import { easAttestationQuery, getEASAttestationQueryVariables, GetEASAttestationQueryResponse, EASAttestationsQueryVariables } from '../queries/easAttestations';
 import { isChainSupported, easSupportedChains } from '../utils/easAttestation';
 import { EASAttestation, EASSchemaUid } from './types';
 import type { Address, Chain } from 'viem';
 
 export type GetEASAttestationsOptions = {
```

Comment:
Let's have the types in the `types.ts` file.

Reply:
I like your thinking what if we do:
- if the type is exported goes into `src/core/types.ts`

then we follow yours
- if the type is used in more than one file, I move it to the shared type file
- if the type is used only in the file, I keep in the file